packages/block-editor/src/components/iframe/index.js

```diff
@@ -272,6 +272,7 @@ function Iframe( {
 <html>
 	<head>
 		<meta charset="utf-8">
+		<base href="${ window.location.origin }">
```

Comment:
Does this work, regardless of how the site is configured (top level, sub folder, ...) and also for third-party block editors?

Reply:
This shouldn't matter for relative urls starting with `/`, they should always use the root.
But for relative urls that don't start with a slash it's trickier. We'd have always use the current post URL etc. This wouldn't work without an iframe either (in the admin), so not sure if it's worth fixing.